docs/reference/expectations/result_format.md

```diff
@@ -4,66 +4,62 @@ title: Result format
 
 
 The `result_format` parameter may be either a string or a dictionary which specifies the fields to return in `result`.
-  * For string usage, see `result_format` values.
+  * For string usage, see [`result_format` values and fields](#result_format-values-and-fields).
```

Comment:
Nit - What about limiting the link to the text after the inline code snippet? It just gets rendered a little wonky (with the underline and the underscore of `result_format`. 
![image](https://user-images.githubusercontent.com/9903066/211339087-d3696bec-96e6-4b58-ae4b-a405b5af800e.png)

```suggestion
  * For string usage, see `result_format` [values and fields](#result_format-values-and-fields).
```